ui/pages/create-account/connect-hardware/index.js

```diff
@@ -158,7 +161,10 @@ class ConnectHardwareForm extends Component {
           });
         } else if (
           errorMessage !== 'Window closed' &&
-          errorMessage !== 'Popup closed'
+          errorMessage !== 'Popup closed' &&
+          !errorMessage
```

Comment:
This can be kind of confusing. Maybe instead we do:

```
errorMessage.toLowerCase().includes('KeystoneError#sync_cancel'.toLowerCase() === false
```